docs/data/material/getting-started/templates/dashboard/components/MenuContent.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import Stack from '@mui/material/Stack';
+import HomeRoundedIcon from '@mui/icons-material/HomeRounded';
+import AnalyticsRoundedIcon from '@mui/icons-material/AnalyticsRounded';
+import PeopleRoundedIcon from '@mui/icons-material/PeopleRounded';
+import AssignmentRoundedIcon from '@mui/icons-material/AssignmentRounded';
+import SettingsRoundedIcon from '@mui/icons-material/SettingsRounded';
+import InfoRoundedIcon from '@mui/icons-material/InfoRounded';
+import HelpRoundedIcon from '@mui/icons-material/HelpRounded';
+
+const mainListItems = [
+  { text: 'Home', icon: <HomeRoundedIcon /> },
```

Comment:
@michaldudak I'm curious, do you see any advantage of the `icon` values being instances (`<HomeRoundedIcon />`) vs. a component (`HomeRoundIcon`) and instantiating it in `MenuContent`'s return below? Is there a significant difference?

Reply:
Not really. I think it's a matter of preference in this case. The current approach allows to pass in props at the point of declaring the variables (but it's not used here, so 🤷)

Reply:
Thanks for the clarification :blush